internal/controller/components/modelregistry/modelregistry_support.go

```diff
@@ -38,6 +38,7 @@ var (
 		"IMAGES_MODELREGISTRY_OPERATOR": "RELATED_IMAGE_ODH_MODEL_REGISTRY_OPERATOR_IMAGE",
 		"IMAGES_GRPC_SERVICE":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
 		"IMAGES_REST_SERVICE":           "RELATED_IMAGE_ODH_MODEL_REGISTRY_IMAGE",
+		"IMAGES_OAUTH_PROXY":            "RELATED_IMAGE_OSE_OAUTH_PROXY_RHEL9_LATEST_IMAGE",
```

Comment:
i will let @dhirajsb  decide how this is going to be. 
ref https://issues.redhat.com/browse/RHOAIENG-23542 
but preferably,  such change should come from component team.

Reply:
We had considered this approach earlier @zdtsw but we were thinking of fixing it in code in RHDS. But, looks like @riprasad wants to standardize the image variable across components.

Reply:
yep. 
my point is, if we get this out, do we still need the change in ModelReg Operator from https://issues.redhat.com/browse/RHOAIENG-23542 ?

Reply:
@zdtsw agreed, this does override the change introduced in the rhds repo. At the very least it lets our QE test against the latest productized proxy image.